Gemfile

```diff
@@ -4,3 +4,7 @@ gemspec
 
 plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
 eval_gemfile(plugins_path) if File.exist?(plugins_path)
+
+gem "rubocop-rake", "~> 0.6.0"
+
+gem "rubocop-rspec", "~> 2.4"
```

Comment:
why do we need these?
`rubocop` shouldn't be needed for the users, right?

Reply:
No, this is probably the wrong place. Should be in `gemspec` as a `development_dependency`